### ISSUE TITLE
checked.t hangs on Windows

### ISSUE THREAD

**Issue by chorny** [xenoterracide/Dist-Zilla-Plugin-Test-PodSpelling#7]
Nothing is printed.
Strawberry 5.14.0, 5.8.9.

**Comment by xenoterracide** [comment]
it fails on line #59, no idea why.

**Comment by xenoterracide** [comment]
although some playtest code shows a readline error on unopened file if I try to slurp it with path class.

**Comment by xenoterracide** [comment]
https://rt.cpan.org/Ticket/Display.html?id=82173 RT Dupe, but has more info.

**Comment by xenoterracide** [comment]
can you please confirm that the workaround in    2.002006  is acceptable

**Comment by rwstauner** [comment]
Test::Spelling 0.17 was released which now works with windows, however in attempting to patch this dist
I'm now getting bizarre "permission denied" errors on windows when `File::Temp` is trying to clean up the `xt/author/pod-spell.t` file that dzil writes.

`\x{1F62D}`

Even dumber is that i can manually do `perl -e unlink` and it works just fine, but for some reason _inside_ the original process there's a permission error.  I've tried a dozen incantations so far with no luck, but will try again when i have time.
